### ISSUE TITLE
[BUG] All Amiga games crash upon starting on Ubuntu 20.04.1 LTS x86_64

### ISSUE THREAD

**Issue by bvrielink** [libretro/libretro-uae#327]
When starting any Amiga game it crashes immediately back to the Retropie screen after trying to start the game. This is on Ubuntu 20.04.1 LTS x86_64 with RetroPie. Everything is updated to the latest versions and lr-puae is installed from source via the experimental packages in Retropie. Not related of course but SNES, Megadrive, N64 and PS1 games are running with no problem on this laptop via Retropie. Kickstart roms are in the BIOS directory and named as required.

Example error in the /var/log/kernel.log:

kernel: [11737.350111] traps: retroarch[68751] general protection fault ip:7f3227188ebe sp:7fff720ed5b0 error:0 in puae_libretro.so[7f322716a000+539000]

Example error in runcommand.log:

/opt/retropie/supplementary/runcommand/runcommand.sh: regel 1264: 68751 Segmentatiefout         (geheugendump gemaakt) /opt/retropie/emulators/retroarch/bin/retroarch -L /opt/retropie/libretrocores/lr-puae/puae_libretro.so --config /opt/retropie/configs/amiga/retroarch.cfg "/home/user/RetroPie/roms/amiga/game.lha" --appendconfig /dev/shm/retroarch.cfg

Can upload a core dump if needed but it's quite big and doesn't offer a lot of information from what I can tell.

If you need more information or want me to test something just let me know.

**Comment by sonninnos** [comment]
Please provide a verbose RetroArch log instead. Are you sure the Kickstart md5s match? And is the core version really the latest one according to this repo?

**Comment by bvrielink** [comment]
Double checked and Kickstart ms5's are matching. Retropie does a git clone from master so it's the latest official version. I am not familiar with Retroarch (as I am using the Retropie shell around it). Do you want me to do something in Retroarch or you just want me to open and close it in verbose mode?

**Comment by sonninnos** [comment]
Ok, darn. Either add `--verbose` to the command or temporarily enable it in the menus.

**Comment by bvrielink** [comment]
As requested the log file. This comment in the log file made me smile btw: Your Amiga program just did something terribly stupid.

[retroarch.log](https://github.com/libretro/libretro-uae/files/5417951/retroarch.log)

**Comment by sonninnos** [comment]
`[libretro INFO] Unknown ROM '/home/user/RetroPie/BIOS/kick40063.A600' loaded`

Just as I suspected, the A600 Kickstart is not correct. Also it is possible that the particular game is one of those that won't work currently even with the correct Kickstart.

Also after you have the correct file you must delete the WHDLoad-directory and let the core create it again under RetroArch `saves` now that it has been populated with the wrong file.

**Comment by bvrielink** [comment]
My mistake. I checked all roms twice but somehow missed this one. It was a valid rom btw (same version, machines etc but different md5). Also this rom worked correctly with Amiberry. Anyway, would be nice if it would output an error to the screen instead of segfaulting but that is for you guys to decide. Replaced the rom and checked a couple of games and it works ok now. So will close this issue. Thanks!
